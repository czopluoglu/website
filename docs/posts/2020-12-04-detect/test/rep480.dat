00100111011010110101
10001101111011110111
11100001101111011111
11001111111110101100
10000101101011111110
10111111111110110010
10001111111011101111
01101110110010101111
10011111111110110110
11001100101100111111
01110000111110111011
00010001101011100000
00111100001001101010
10001011111101101111
00100001101010101010
00101001001101111110
01111001001100100110
11110100111110001110
00000010011110100110
10001100001111111101
10000000110110100101
10110101001001000010
11111111111111101001
10000000011111101101
00110100001010100011
11010001001000000011
11101000001011110110
11000011011100110110
00010000001111001000
10011101011111011111
11110101110010110111
10010111100100100010
00000001111000010010
00111001110110111010
00000000000000000000
00100101000001100110
10000101110011011111
10000000001010000001
10100000111111111111
11110001111111101010
11000101000010100100
11100111111100100111
10000100001011000011
10010000010000110010
10001101001100101110
10000001100100100110
11011111110110100011
11101111011100111111
11100010011010100000
11000001000010101100
11110000101001011000
11101111111110111111
11110110111010111100
11011111111001111011
11001001110010110010
00110001111101100110
11111011110110110110
11000111111010001010
00000010000000000000
11001011111101111111
11010111101011110101
10010111111100111011
11100000011111111110
01101000001110100010
00100011111110011011
11010110111011100011
11111101111010011110
11100111110101111111
10011101111111101111
01001110011111111110
00101110110111011110
11011000000100111011
00101111101010111101
10100111111101101110
00000001111100100011
10010001111111101101
11011000111000001011
11111111011001010110
11100101101110011111
01110110110010110011
01100011000000100001
10000011001110111001
00010101111111111111
11100111100011111111
10100010111111000111
01100010101000000010
11010011111110111011
10110101110010010000
00110101101111100111
01110011000001110100
00011001101000000101
11011001101110110010
01011001011101101111
00011101001101111011
00110011100010010110
11000001011110110110
10100000010111111110
01000001011010111100
01010110111110101010
10001111011101111111
11110100111001100010
01111111110110101110
10111101111011111111
10110101111111111110
10000001001111101111
10100100001001011011
00111001011011011110
11011101111111110111
00111101010011100011
10011000101011000011
01001000100100001111
11010101111111101111
01010111011011111110
00001101101111011111
11110000010001011111
00100100000010001111
01010100001100101110
00000000111111111111
11011101011011100111
00110100100101000010
11001000001001100001
00000111001101101110
00100100101010011111
11100001000110100011
10000000111000101110
11001111111111111111
11000001000100110011
10101101001001101111
10000000011010001111
11000101001100111111
01100000000000000001
10111101101011101101
00011101011101010100
01100000011111000010
10011010100110000110
00111111111011010001
01000111110100000011
11010000111111001110
10001101111010111101
10000000011010100001
10101001011010011111
10000000010100100111
01101111110110101011
10111001101100101010
11000010110000101100
00000001111111111111
01011101001000110011
11100000100000000010
10000000011010001110
01111111111110100111
01010010100010110001
10011101011111110100
10110101010110011111
11100001110011110111
01000101010011101110
00101101111100001000
11111000101011000110
10010100001000000011
00000001001000000101
10001001110010100110
10000001010001001000
10000010111111101110
00001000000101000000
00011111110010000111
10111101110110111011
10110111101010000110
10111011111010011101
10011011011001010101
11000100110001000100
10110110001100110001
11110100111100000010
01000000011010010011
10000110000000000001
10101000001111011111
01000101100000011111
10011001101111100110
01001100011100001010
10001011001101000010
00010000011110000110
00001000111000100010
10011100011001001100
00110011010010111011
11010101011101111111
10000000101011001110
11111011111010111110
01011010001011111110
11010000111111101110
01000001001100000000
00100111111000000100
10010100111000010111
11110100011110100110
10000101011111101101
00000101110100101011
11100001111111111101
00000011111110111110
11011111111011000010
01111101111000101010
10111101001010000110
11011101011100111111
10011011111011111111
11010001101111001110
10111111111100110011
00101001000110011000
11110101111000101101
01011001110001001101
00011101111001111100
00010001111011111011
01011111011111111111
01010001101110100010
10111110011111100111
10011100111010011010
10000101101110101111
01110001011011111111
00000000111001000111
01111101000111111111
10001001001011101110
10000001100001000111
01010001010100100010
01100000001000110111
00001101000000110011
01101101011111000110
01011001100111010001
00010011111010100101
11000001000011000010
10011011110110111110
10011101011111101000
01010111111110100010
00101001011110010111
10011001110111000001
11001101001000100110
01101000010000101011
00100001110111111111
11110001111100101011
11101101100110110111
10111101111000000110
00101111001010111110
00000000000000101011
01000111101111011010
01111000011111000110
10001001001001000110
01100111011110001111
01111000001000101011
00000000011100101001
11001111000001111110
10101001001000101011
00111001100001001001
10101011001100100101
01101000001110000010
01011111111001100110
01011100110110001110
