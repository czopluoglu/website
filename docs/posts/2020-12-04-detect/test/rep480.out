-------------------------------------------------------
                  DETECT SUMMARY OUTPUT
-------------------------------------------------------
 
                    Data File Name: C:/Users/cengiz/Desktop/Github/website/_posts/2020-12-04-detect/test/rep480.dat                                         

              Number of Items used:       20

           Number of Items dropped:        0

               Number of Examinees:      250

                 Minimum Number of 
                Examinees per Cell:        2

         Number of Vectors Mutated:        4

      Maximum Number of Dimensions:       12

                Randomization Seed:    99991

   Minimum percentage of examinees
         used after deleting cells
     having less than  2 examinees:    97.20

   Average percentage of examinees
         used after deleting cells
     having less than  2 examinees:    99.08
 
-------------------------------------------------------

 NUMBER OF DIMENSIONS THAT MAXIMIZE DETECT:  4
 
  Exploratory DETECT Statistics:

              Maximum DETECT value:   0.7545
                   IDN index value:   0.7158
                           Ratio r:   0.6273

 PARTITION WITH MAXIMUM DETECT VALUE:

    1    2    3    4    4    4    4    4    4    4
    1    2    3    1    2    3    1    1    2    3

 CLUSTER MEMBERSHIPS:

   -----------CLUSTER  1-------------

    1   11   14   17   18

   -----------CLUSTER  2-------------

    2   12   15   19

   -----------CLUSTER  3-------------

    3   13   16   20

   -----------CLUSTER  4-------------

    4    5    6    7    8    9   10
 
   ----------------------------------
 
  Covariance Sign Pattern Matrix:
   d++-+-------+-----+-
   +d+++-+------+++----
   ++d++-----+--++-----
   -++d+-++---++------+
   ++++d-----++---+-+-+
   -----d++++---+--+---
   -+-+-+d++-+--------+
   ---+-++d---++---++--
   -----++-d+----++--+-
   -----+--+d++++-++---
   --+-+-+--+d+-+--+-++
   ---++--+-++d+---++--
   +--+---+-+-+d--+-+--
   -++--+---++--d++--++
   -++-----+----+d+-+--
   -+--+---++--+++d++--
   -----+-+-+++---+d+++
   ----+--+---++-+++d+-
   +-------+-+--+--++d+
   ---++-+---+--+--+-+d
 
 No cross validation for this DETECT run
